packages/api/src/beacon/routes/validator.ts

```diff
@@ -486,7 +486,7 @@ export type ReqTypes = {
     query: {
       randao_reveal: string;
       graffiti: string;
-      skip_randao_verification?: boolean;
+      skip_randao_verification?: string;
```

Comment:
what about the schema, I don't see this being changed by this PR

https://github.com/ChainSafe/lodestar/blob/9b1f0af945ad7405ae3938178b35e99d969abdf7/packages/api/src/beacon/routes/validator.ts#L585